civis/resources/_resources.py

```diff
@@ -411,13 +421,14 @@ def parse_method(verb, operation, path):
     summary = operation["summary"]
     params = operation.get("parameters", [])
     responses = operation["responses"]
-    deprecated = operation.get('deprecated', False)
-    if 'deprecated' in summary.lower() or deprecated:
+    deprecation_warning = operation.get("x-deprecation-warning", None)
+    if 'deprecated' in summary.lower():
```

Comment:
I have to leave this in because there seem to be a handful of endpoints that use this paradigm. We can get rid of this once we remove those endpoints.

Reply:
Strange that we have endpoints which explicitly do not work but are marked as deprecated. This seems like a reasonable workaround in the meantime.